lectures/heavy_tails.md

```diff
@@ -1020,9 +1024,9 @@ Since $r \geq \alpha$, we have $\mathbb E X^r = \infty$.
 ```{exercise}
 :label: ht_ex3
 
-Repeat exercise 1, but replace the three distributions (two normal, one
-Cauchy) with three Pareto distributions using different choices of
-$\alpha$.
+Refer to the figure in the section titled "Simulations". 
```

Comment:
@chappiewuzefan can we use a figure label and provide a link to the figure mentioned here?

Reply:
fixed, but there are a very strange behaviour in jupyter lab
when I run a block of code, the block is disappeared:
<img width="1019" alt="image" src="https://github.com/QuantEcon/lecture-python-intro/assets/37008392/8437a11f-5c24-4151-ba12-dad356464da3">
<img width="1027" alt="image" src="https://github.com/QuantEcon/lecture-python-intro/assets/37008392/0d84ae11-6b39-45e6-a9de-f0c6a352ab05">

Reply:
@mmcky this behaviour let me can't see the block .... I have no idea why is that